src/components/smart_objects/src/object_schema_item.cc

```diff
@@ -157,6 +158,39 @@ errors::eType CObjectSchemaItem::validate(
   return errors::OK;
 }
 
+bool CObjectSchemaItem::filterInvalidEnums(
+    SmartObject& Object,
+    const utils::SemanticVersion& MessageVersion,
+    rpc::ValidationReport* report) {
+  bool valid = true;
+  for (const auto& key : Object.enumerate()) {
+    std::map<std::string, SMember>::const_iterator members_it =
```

Comment:
good place for auto

Reply:
Fixed in https://github.com/smartdevicelink/sdl_core/pull/3385/commits/e547238c73dc223287a8ac46cfbe6a6e92e3c18c